### ISSUE TITLE
train.py: error: argument --loss-type: invalid int value: 'arcface'

### ISSUE THREAD

**Issue by deimsdeutsch** [deepinsight/insightface#1235]
```
CUDA_VISIBLE_DEVICES='0' python3 -u train.py --network vargfacenet --loss arcface --dataset retina
usage: train.py [-h] [--data-dir DATA_DIR] [--prefix PREFIX]
                [--pretrained PRETRAINED] [--ckpt CKPT] [--network NETWORK]
                [--version-se VERSION_SE] [--version-input VERSION_INPUT]
                [--version-output VERSION_OUTPUT]
                [--version-unit VERSION_UNIT] [--version-act VERSION_ACT]
                [--end-epoch END_EPOCH] [--noise-sgd NOISE_SGD] [--lr LR]
                [--wd WD] [--mom MOM] [--emb-size EMB_SIZE]
                [--per-batch-size PER_BATCH_SIZE] [--margin-m MARGIN_M]
                [--margin-s MARGIN_S] [--margin-a MARGIN_A]
                [--margin-b MARGIN_B] [--easy-margin EASY_MARGIN]
                [--margin-verbose MARGIN_VERBOSE]
                [--logits-verbose LOGITS_VERBOSE]
                [--c2c-threshold C2C_THRESHOLD] [--c2c-mode C2C_MODE]
                [--output-c2c OUTPUT_C2C] [--train-limit TRAIN_LIMIT]
                [--margin MARGIN] [--beta BETA] [--beta-min BETA_MIN]
                [--beta-freeze BETA_FREEZE] [--gamma GAMMA] [--power POWER]
                [--scale SCALE] [--center-alpha CENTER_ALPHA]
                [--center-scale CENTER_SCALE]
                [--images-per-identity IMAGES_PER_IDENTITY]
                [--triplet-bag-size TRIPLET_BAG_SIZE]
                [--triplet-alpha TRIPLET_ALPHA]
                [--triplet-max-ap TRIPLET_MAX_AP] [--verbose VERBOSE]
                [--loss-type LOSS_TYPE] [--incay INCAY]
                [--use-deformable USE_DEFORMABLE] [--rand-mirror RAND_MIRROR]
                [--cutoff CUTOFF] [--patch PATCH] [--lr-steps LR_STEPS]
                [--max-steps MAX_STEPS] [--target TARGET]
train.py: error: argument --loss-type: invalid int value: 'arcface'

```

**Comment by deimsdeutsch** [comment]
Cannot use python3
